ngraph.mk

```diff
@@ -67,7 +67,7 @@ endif
 
 NGRAPH_SRC_DIR := $(ROOTDIR)/3rdparty/ngraph
 NGRAPH_BUILD_DIR := $(ROOTDIR)/3rdparty/ngraph/build
-NGRAPH_INSTALL_DIR := $(ROOTDIR)/3rdparty/ngraph/install
+NGRAPH_INSTALL_DIR := $(ROOTDIR)/3rdparty/ngraph/build
```

Comment:
I'm mildly against this change.  IMO the layout of the "build" tree is kind of an internal implementation detail of ngraph and subject to change without notice; the "install" directory's layout is more like a public interface.  So I'd prefer to retain that distinction.

But if you feel strongly about the need for this, and it works in practice, I'll roll with it.

Reply:
with `install` , after we build mxnet, it shows `3rdparty/ngraph/` as modified/dirty in git status, as `install` is not in mxnet .gitignore; we could pick another subdir, or modify .gitignore. The above change works as well.